packages/client/bin/cli.ts

```diff
@@ -395,13 +395,13 @@ async function startClient(config: Config, customGenesisState?: GenesisState) {
 
   let blockchain
   if (customGenesisState !== undefined) {
-    const validateConsensus = config.chainCommon.consensusAlgorithm() === ConsensusAlgorithm.Clique
+    //const validateConsensus = config.chainCommon.consensusAlgorithm() === ConsensusAlgorithm.Clique
     blockchain = await Blockchain.create({
       db: dbs.chainDB,
       genesisState: customGenesisState,
       common: config.chainCommon,
       hardforkByHeadBlockNumber: true,
-      validateConsensus,
+      validateConsensus: true,
```

Comment:
Unsure: why did we only validate consensus on Clique previously?

Reply:
Because PoW is slow 🙃.

Reply:
I also thought this, but @g11tech thought otherwise :thinking: